test/db/index.test.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs'
```

Comment:
Thank you for including tests!
But I would stay this is testing more the general environment that the actual business logic or data processing functions you have here...

Reply:
Yes they are, I wrote this just to get started. Do you want a full coverage test suite? I could do it :)